que_.py

```diff
@@ -0,0 +1,26 @@
+from doubly_linked_list import DoublyLinkedList
+
+class Queue:
+    def __init__(self, iterable=None):
+        self.dll = DoublyLinkedList(iterable)
+
+    def enqueue(self, value):
+        self.dll.push(value)
+
+    def dequeue(self):
+            try:
+                return self.dll.shift()
+            except:
+                raise ValueError("Dequeue does not work on an empty queue")
```

Comment:
bare excepts are not allowed, Exactly which exception do you expect here?

Maybe like this?
```suggestion
            except ValueError:
                raise ValueError("Dequeue does not work on an empty queue")
```

Reply:
Yeah I see the error there, I didn't catch that as the tests still passed.

Reply:
Python doesn't require a specific error.  But there is value in only allowing the specific errors you expect to be caught and handled.